test/cypress/integration/36-pledges.text.js

```diff
@@ -10,7 +10,7 @@ describe('Pledges', () => {
       cy.get('[data-cy=githubHandleInput]').clear().type('opencollective/opencollective-frontend');
       cy.get('[id=publicMessage]').clear().type('publicMessage');
       cy.get('[data-cy="submit"]').click();
-      cy.get('[data-cy="currencyAmount"]').should('contain', '$');
+      cy.get('[data-cy="currencyAmount"]').should('be.visible').should('contain', '$');
```

Comment:
`should('contain', '$')` will not pass unless the element is visible. All commands have the same timeout, so I don't get why we need these `.should('be.visible')`

Reply:
I noticed that cypress was unable to locate the element and so i went through this https://github.com/cypress-io/cypress/issues/695 and found a solution which was working locally for me in headless mode as well so I assumed be.visble should be a fix that could help solve the issue of failure

Reply:
Cypress being unable to locate the element is probably not a bug, and just means that the element is not on the page. Either because the submit failed, or other reasons.  

Anyway the issue you're describing is still there, even with the `shoud('be.visible')`: https://github.com/opencollective/opencollective-frontend/pull/3803/checks?check_run_id=551411901.

Reply:
This may be a failure on API side, let me investigate

Reply:
so would you like me to remove be.visble ??

Reply:
Yes, in any case I still don't see why we would need it

Reply:
will do it right away